geeksurvey/views.py

```diff
@@ -109,6 +112,146 @@ def profile_update(request):
 
     return render(request, 'profile/update.html', context)
 
+@login_required
+def profile_fund(request):
+    payment = Payment(owner=request.user, amount=125)
+    payment.save()
+    request.session['payment-id'] = str(payment.id)
+
+    profile = Profile.objects.get(user=request.user)
+
+    paypal_dict = {
+            # TODO get this from decouple config
+            "business": config('PAYPAL_BIZ_EMAIL_P'),
+            "amount": payment.amount,
+            "currency_code": "USD",
+            "item_name": 'Fund GeekSurvey Account',
+            "invoice": str(payment.id),
+            "notify_url": request.build_absolute_uri(reverse('paypal-ipn')),
+            "return_url": request.build_absolute_uri(reverse('paypal-return')),
+            "cancel_return": request.build_absolute_uri(reverse('paypal-cancel')),
+            "lc": 'EN',
+            "no_shipping": '1',
+        }
+
+    form = PayPalPaymentsForm(initial=paypal_dict)
+    context = {
+        'profile': profile,
+        'form': form,
+        }
+
+    return render(request, 'profile/fund.html', context)
+
+@login_required
+def profile_claim(request):
+    # GET:
+    # claim.html contains a form where
+    # the user enter their paypal email address
+    # -
+    # POST:
+    # if valid email address, send payment to paypal
+    # from BIZ to user_addr
+    # of amount profile.balance
+    # in IPN, set their balance to 0
```

Comment:
not in IPN, we just do that here in the same function since these payments always go through on sandbox